packages/cli/src/lib/__tests__/fixtures/merge/i18nIntoBase/expected.jsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react'
 import { I18nextProvider } from 'react-i18next'
 import i18n from 'web/src/i18n'
+
+/** @type { import("@storybook/csf").GlobalTypes } */
```

Comment:
We don't make extensive use of the code being tested here so the change is ok to get Babel in. At least we're testing for it—it just doesn't handle comments well it looks like in terms of deduplication.